src/core/positioning/nmeagnssreceiver.h

```diff
@@ -51,10 +51,38 @@ class NmeaGnssReceiver : public AbstractGnssReceiver
     void handleStartLogging() override;
     void handleStopLogging() override;
 
+    void processImuSentence( const QString &sentence );
+
     QTime mLastGnssPositionUtcTime;
 
     QFile mLogFile;
     QTextStream mLogStream;
+
+    GnssPositionInformation mCurrentNmeaGnssPositionInformation;
+
+    struct ImuPosition
+    {
+        bool valid = false;
+        QDateTime utcDateTime;
+        double latitude;
```

Comment:
I'd initiate all of those doubles to std::numeric_limits<double>::quiet_NaN();

Reply:
Done